### DISCUSSION TITLE
How to make ripgrep display the completed content of the searched results

### DISCUSSION THREAD

**Discussion by khailifz** [BurntSushi/ripgrep#2688] [Q&A] [answered]
when i use ripgrep to find ,it return to me 
`tar_logcat.log.09/tar_logcat/logcat.log.037:12-12 01:22:16.037  4757  4790 I TspManager: postTrackingData jsonMessage:{"commondata":{"app_package_name **[... 0 more matches]**
`
but i want result show like grep tool like this 
`tar_logcat.log.01/tar_logcat/logcat.log.043:2259:12-12 07:16:48.011  4757  4802 I TspManager: postTrackingData jsonMessage:{"commondata":{"app_package_name":"com.app.launcher","app_starttime":"","app_version":"14","behaviorid":"","clienttype":"","clientversion":"V1.0.6.14","cp_source":"com..app.launcher","device_starttime":"","event_type":"","frombrand":"","happentime":"12 07:16:48","id":"","imei":"","imsi":"","interactiveway":"1","network_type":"1","phone_num":"","screen_ratio":"16:9","screen_size":"1280x720","sign":"","userid_deviceid":"0\","uuid":"","vin":""},"trackingdata":{}}
`

**Comment by BurntSushi** [comment, marked as answer]
The `[... 0 more matches]` is appearing because ripgrep is trimming a long line. This is _not_ default behavior. You had to opt into it. Either you're passing the `-M/--max-columns` flag explicitly, or you have it in an alias, shell function, wrapper script or a ripgrep config file.

If you re-run your command with `--debug`, then it should show any arguments loaded from a config file. Otherwise, try running `which rg` to see whether it's an alias or a shell function or a wrapper script that's inserting the extra argument.
